### PR TITLE
add tini to commercial QR - usage in fedramp env for a month+ 

### PR DESCRIPTION
**Pull request by jonmosco**
Issues with running QR with worked out deploying this in FedRAMP for over a month.  

https://issues.redhat.com/browse/APPSRE-9812

### CODE REVIEW

**Comment by jonmosco** [comment]
Also added comments for better visual separation of build layers.

**Comment by TGPSKI** [comment]
@jonmosco can you expand on what tini is, what value we get out of adding it to the container, the problem it solved in FedRAMP, and how this could be beneficial to the commercial environment?

File: dockerfiles/Dockerfile [hunk 1]
```diff
@@ -43,40 +48,48 @@ USER reconcile
...
 FROM prod-image as fips-prod-image
 ENV OC_VERSION=4.10.15
 
 # Tini
-ENV TINI_VERSION v0.19.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
-RUN chmod +x /tini
+# ENV TINI_VERSION v0.19.0
+# ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+# RUN chmod +x /tini
```
Comment (hemslo) [inline dockerfiles/Dockerfile:89]: let's remove unused comments

**Review by hemslo — APPROVED** [review]
LGTM! Zombie Reaper!

**Review by rporres — APPROVED** [review]

**Comment by cubismod** [comment]
@jonmosco just to check, did you do a run of the docker image with a QR integration? Ik we encountered issues with entrypoints before so just want to be 100% sure we don't run into those here

**Review by cubismod — APPROVED** [review]
LGTM

**Comment by TGPSKI** [comment]
LG 
TM

**Pull request merged**